tests/Integration/SDK/SpanBuilderTest.php

```diff
@@ -230,6 +236,62 @@ public function test_set_attribute_after_end(): void
         $span->end();
     }
 
+    public function test_set_attribute_null_string_value(): void
+    {
+        /** @var Span $span */
+        $span = $this
+            ->tracer
+            ->spanBuilder(self::SPAN_NAME)
+            ->setAttribute('nil', null)
+            ->setAttribute('empty-string', '')
+            ->startSpan();
+
+        $attributes = $span->toSpanData()->getAttributes();
+        $this->assertSame(1, $attributes->count());
+        $this->assertSame('', $attributes->get('empty-string'));
+        $this->assertNull($attributes->get('nil'));
+
+        $span->end();
+    }
+
+    public function test_set_attribute_only_null_string_value(): void
```

Comment:
this test could have a more descriptive name (they can be really long, that doesn't matter). test_set_attribute_with_only_null_string_should_<what should it do?>